src/software/ai/hl/stp/tactic/penalty_kick_tactic/penalty_kick_tactic_fsm.h

```diff
@@ -0,0 +1,299 @@
+#pragma once
```

Comment:
can we move the penalty kick files to a `tactic/penalty_kick` folder?

Reply:
done